### DISCUSSION TITLE
Topology PNG download always dark background?

### DISCUSSION THREAD

**Discussion by MrDragon** [mattieserver/netbox-topology-views#423] [Q&A] [answered]
Ladies and Gents,

I'm currently running NetBox 3.6.4, but I've had that issue before. When I download a topology PNG, it's always on a dark background, regardless of my theme setting.
I have many "black" cables, and the device names also show in black on the PNG.
How/where can I set the background color for downloaded PNGs?
What am I missing?
I'm not afraid to go into the code and change a color variable, but I don't even know where to begin.

Thanks for any pointers!

   Frank

**Comment by mattieserver** [comment, marked as answer]
Hi,

I can't check right now but I think the background might be transparent. So it might be the OS theme that makes it look dark.


________________________________
From: Frank Mogaddedi ***@***.***>
Sent: Wednesday, November 8, 2023 5:40:51 PM
To: mattieserver/netbox-topology-views ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [mattieserver/netbox-topology-views] Topology PNG download always dark background? (Discussion #423)


Ladies and Gents,

I'm currently running NetBox 3.6.4, but I've had that issue before. When I download a topology PNG, it's always on a dark background, regardless of my theme setting.
I have many "black" cables, and the device names also show in black on the PNG.
How/where can I set the background color for downloaded PNGs?
What am I missing?
I'm not afraid to go into the code and change a color variable, but I don't even know where to begin.

Thanks for any pointers!

Frank

**Comment by MrDragon** [comment]
Yes, Clearly ID-10-T issue (be that on my chair or at Micro$oft)
Default opens with MS photos or whatever that abysmal thing is called.
I apologize for everything,,,
